Projects/Foundations/Schemas/Docs-Books/community-data-mines.json

```diff
@@ -56,14 +56,18 @@
         {
             "style": "Chapter",
             "text": "X->Foundations->Topic->Sparta Data Mine->"
+        },
+		{
+            "style": "Chapter",
```

Comment:
@quantum8 If you have a moment, could you move the opening bracket  to the left so it lines up with the closing brakcet above? It's not important for the functionality but the structure.